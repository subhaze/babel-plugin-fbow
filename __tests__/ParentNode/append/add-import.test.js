const babel = require("babel-core");
const plugin = require("../../../index.js");

const testCode = `
var parent = document.createElement("div");
var p = document.createElement("p");
var p2 = document.createElement("p");
parent.append(p);
parent.append(p2);
`;

it("adds import for append once", () => {
	const { code } = babel.transform(testCode, {
		plugins: [plugin],
	});
	expect(code).toMatchSnapshot();
});
