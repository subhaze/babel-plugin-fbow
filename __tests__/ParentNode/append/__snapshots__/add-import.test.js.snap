// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`adds import for append once 1`] = `
"import \\"fbow/ParentNode/append\\";

var parent = document.createElement(\\"div\\");
var p = document.createElement(\\"p\\");
var p2 = document.createElement(\\"p\\");
parent.append(p);
parent.append(p2);"
`;
